### PR TITLE
Pset_LateralExcavationCommon

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_LateralExcavationCommon

## Description: 

Common properties for lateral excavation elements

| Property                  | Datatype                        | Description                                                  |
| ------------------------- | ------------------------------- | ------------------------------------------------------------ |
| RadialExcavationType      | PEnum_RadialExcavationType      | Type of radial excavation                                    |
| RadialExcavationTypeOther | IfcLabel                        | A user defined type of radial excavation when RadialExcavationType = OTHER |
| RadialExcavationDirection | PEnum_RadialExcavationDirection | Direction of radial excavation from main tunnel drive        |
| RadialExcavationDepth     | IfcPositiveLengthMeasure        | Depth of radial excavation                                   |

## Enumeration: PEnum_RadialExcavationType

| Constant           | Description |
| ------------------ | ----------- |
| PROFILEENLARGEMENT |             |
| REPROFILING        |             |
| NICHE              |             |
| OTHER              |             |

## Enumeration: PEnum_RadialExcavationDirection

| Constant | Description |
| -------- | ----------- |
| LATERAL  |             |
| VERTICAL |             |
| RADIAL   |             |

### CODE REVIEW

File: IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Direction of radial excavation from main tunnel drive.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/Documentation.md]:
```suggestion
Direction of radial excavation relative to the main tunnel drive.
```

File: IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/DocProperty.xml:1]: Still having some issues with the constants. I assume `VERTICAL` means in the direction of gravity, right? I would celebrate some documentation.

**Review by SergejMuhic — COMMENTED** [review]
Constants and enumerations do not have any documentation.

File: IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/r/RadialExcavationDirection_0PWybQDsb7VuLXarptQq0N/DocProperty.xml:1]: This comment should be answered by @JontyDunn

**Pull request merged**